mail_tracking_mailgun/models/res_partner.py

```diff
@@ -34,9 +34,13 @@ def _email_bounced_set(self, reason, event):
                 event.id or 0,
                 event.id or _("unknown"),
             )
-            body = _(
-                "Email has been bounced: %(email)s\nReason: %(reason)s\nEvent: %(event_str)s"
-            ) % ({"email": partner.email, "reason": reason, "event_str": event_str})
+            body = (
+                _(
+                    "Email has been bounced: %(email)s\nReason: "
+                    "%(reason)s\nEvent: %(event_str)s"
+                ),
+                ({"email": partner.email, "reason": reason, "event_str": event_str}),
```

Comment:
This doesn't seem correctly put (with the weird parenthesis), and `_(` accepts already the parameters as new arguments.

Reply:
Changed